app/src/renderer/components/wallet/PageTransactions.vue

```diff
@@ -85,7 +82,8 @@ export default {
     sort: {
       property: "height",
       order: "desc"
-    }
+    },
+    validatorURL: "https://explorecosmos.network/validators"
```

Comment:
How about `${window.location.hostname}/stake/validators`? Will this also open a new window?

Reply:
yup, they both go to `http://localhost:9080/localhost/staking/validators/{address}`

Reply:
OK. let's go with this version first and then switch to vue-router later?